app/javascript/elm/src/Main.elm

```diff
@@ -354,6 +355,7 @@ update msg model =
                     , Cmd.batch
                         [ Ports.toggleSession { deselected = Nothing, selected = Just id }
                         , SelectedSession.fetch model.sensors model.selectedSensorId model.page id (RemoteData.fromResult >> GotSession)
+                        , Ports.highlightSessionMarker Nothing
```

Comment:
Could you explain why this and the next are needed? Shouldn't they disappear onMouseLeave no matter what?

Reply:
When a user clicks on the tile to select the session the tile disappears but the `onMouseLeave` event is not triggered by this.